src/ape_cache/_cli.py

```diff
@@ -0,0 +1,47 @@
+import click
+import pandas as pd
+
+from ape.utils import ManagerAccessMixin
+
+from . import models
+
+
+def get_engine():
+    return ManagerAccessMixin.query_manager.engines["cache"]
+
+
+@click.group(short_help="Query from caching database")
+def cli():
+    """
+    Command-line helper for managing query caching database.
+    """
+
+
+@cli.command(short_help="Initialize a new cache db")
+def init():
+    db_file = get_engine().database_file
+    if not db_file.is_file():
+        click.echo("initializing database")
+        models.Base.metadata.create_all(bind=get_engine().engine)
+        return click.echo("caching database initialized")
```

Comment:
```suggestion
        return click.echo("Caching database initialized")

```

Reply:
Would it be better to use the logger here so we can disable these if we need? `cli_ctx.logger.info()` or something?

Reply:
I will leave that decision to a vote amongst the reviewers, I really don't mind either way

Reply:
Fine to use the logger, not a big deal either way

Reply:
I vote logger! More consistent and then I can disable more easily by changing the log level via `--verbosity`